### ISSUE TITLE
Add back missing shortcuts

### ISSUE THREAD

**Issue by swissspidy** [swissspidy/preferred-languages#1107]
ArrowUp/ArrowDown should just change the selection

Alt+ArrowUp/ArrowDown moves the position
